src/field/captcha/recaptcha.jsx

```diff
@@ -4,6 +4,30 @@ import propTypes from 'prop-types';
 
 const noop = () => {};
 
+const RECAPTCHA_V2_PROVIDER = 'recaptcha_v2';
+const RECAPTCHA_ENTERPRISE_PROVIDER = 'recaptcha_enterprise';
+
+export const isRecaptcha = provider =>
+  provider === RECAPTCHA_ENTERPRISE_PROVIDER || provider === RECAPTCHA_V2_PROVIDER;
+
+const globalForProvider = provider => {
```

Comment:
Let's rename this, how about `getRecaptchaProvider`? As it is, it's a little too ambiguous.